release-notes-1.8.2.md

```diff
@@ -0,0 +1,8 @@
+#### Version Number
+${version-number}
+
+#### New Features
+- SCMOD-12326: Updated to use Artifactory for private Docker images
```

Comment:
This is a patch release, it cannot have any new features. I also would not call this out in the release notes, if you have nothing else to put in here simply add "This release includes OS package updates only.", for example [https://github.com/CAFapi/opensuse-base-image/releases/tag/v1.4.2](https://github.com/CAFapi/opensuse-base-image/releases/tag/v1.4.2)